include/zephyr/drivers/regulator.h

```diff
@@ -242,6 +243,21 @@ static inline bool regulator_common_is_init_enabled(const struct device *dev)
 	return (config->flags & REGULATOR_INIT_ENABLED) != 0U;
 }
 
+/**
+ * @brief Get minimal voltage from common config.
+ *
+ * @param dev Regulator device instance.
+ *
+ * @retval minimal supported voltage from common config.
+ */
+static inline int32_t regulator_get_cfg_min_uv(const struct device *dev)
+{
+	const struct regulator_common_config *cmn =
+		(const struct regulator_common_config *)dev->config;
+
+	return cmn->min_uv;
+}
```

Comment:
```suggestion
/**
 * @brief Get minimum supported voltage.
 *
 * @param dev Regulator device instance.
 * @param min_uv Where minimum voltage will be stored, in microvolts.
 *
 * @retval 0 If successful
 * @retval -ENOENT If minimum voltage is not specified.
 */
static inline int regulator_common_get_min_voltage(const struct device *dev, int32_t *min_uv)
{
	const struct regulator_common_config *config =
		(const struct regulator_common_config *)dev->config;

	if (config->min_uv == INT32_MIN) {
		return -ENOENT;
	}

	*min_uv = config->min_uv;
	
	return 0;
}
```

This avoids exposing common layer internals to drivers (ie INT32_MIN).